common/encode_lavc.c

```diff
@@ -83,25 +83,10 @@ const struct m_sub_options encode_config = {
         {"ovcopts", OPT_KEYVALUELIST(vopts), .flags = M_OPT_HAVE_HELP},
         {"oac", OPT_STRING(acodec)},
         {"oacopts", OPT_KEYVALUELIST(aopts), .flags = M_OPT_HAVE_HELP},
-        {"ovoffset", OPT_FLOAT(voffset), M_RANGE(-1000000.0, 1000000.0),
-            .deprecation_message = "--audio-delay (once unbroken)"},
-        {"oaoffset", OPT_FLOAT(aoffset), M_RANGE(-1000000.0, 1000000.0),
-            .deprecation_message = "--audio-delay (once unbroken)"},
         {"orawts", OPT_BOOL(rawts)},
-        {"ovfirst", OPT_BOOL(video_first),
-            .deprecation_message = "no replacement"},
-        {"oafirst", OPT_BOOL(audio_first),
```

Comment:
however it turns out these didn't do anything anyway

you forgot to remove them from encode.h btw

Reply:
Yeah it hasn't done anything since 6c8362ef54f4e90476553cb6b64996cc414da06d.